packages/shell-chrome/src/backend.js

```diff
@@ -127,10 +128,9 @@ function discoverComponents(isThroughMutation = false) {
         }
 
         const data = Object.entries(rootEl.__x.getUnobservedData()).reduce((acc, [key, value]) => {
-            const type = typeof value
             acc[key] = {
-                value: type === 'function' ? 'function' : value,
-                type,
+                value: value instanceof HTMLElement ? serializeHTMLElement(value) : value,
```

Comment:
we've lost the `'function'` replace here, since we're building a message payload, it would be better to do the function -> `'function'` replace before sending the message (since it will keep the payload smaller)

```suggestion
                value: value instanceof HTMLElement ? serializeHTMLElement(value) : typeof value === 'function' ? 'function' : value,
```

Reply:
If we apply this change, I think we're ready to merge 😄 we can do tests in another PR